addons/sys_intercom/fnc_intercomChildrenActions.sqf

```diff
@@ -22,10 +22,10 @@ private _actions = [];
 private _intercomNames = _target getVariable [QGVAR(intercomNames), []];
 
 {
-    (_intercomNames select _forEachIndex) params ["_intercomName", "_intercomDisplayName"];
+    (_intercomNames select _forEachIndex) params ["_intercomName", "", "_intercomShortName"];
```

Comment:
Personally I would leave as it was before.

Reply:
Previously it was:
```
Main Action:
    Intercom
        Crew Intercom
        Pax Intercom
```
This changes it to:
```
Main Action:
    Intercom
        Crew
        Pax
```
Which I think is better, it's obviously Intercom, you already select that with first action, and it takes less space.

We can of course handle that differently for better compatibility with user-defined intercom names and so on.

Reply:
Let's keep your changes.